services/api-server/src/simcore_service_api_server/api/dependencies/authentication.py

```diff
@@ -1,3 +1,48 @@
-from .auth_basic import get_active_user_email, get_active_user_id
+from fastapi import Depends, HTTPException, Security, status
+from fastapi.security import HTTPBasic, HTTPBasicCredentials
 
-__all__ = ["get_active_user_id", "get_active_user_email"]
+from ...db.repositories.api_keys import ApiKeysRepository
+from ...db.repositories.users import UsersRepository
+from .database import get_repository
+
+# SEE https://swagger.io/docs/specification/authentication/basic-authentication/
+basic_scheme = HTTPBasic()
+
+
+def _create_exception():
+    _unauthorized_headers = {
+        "WWW-Authenticate": f'Basic realm="{basic_scheme.realm}"'
```

Comment:
are you sure about how to capitalize this string? I saw many different ones in the different APIs some times with Www-authenticate, sometimes all lower case, ...

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/WWW-Authenticate